### ISSUE TITLE
New Site: error when importing SearchFilters, store not defined

### ISSUE THREAD

**Issue by jameskirkby** [shiftcommerce/shift-react-components#3]
### Info

Looks like algolia needs to be setup first, possibly something to do with `react-instantsearch` and `react-instantsearch-dom`

```
TypeError: Cannot read property 'store' of undefined
```

### Steps to reproduce:

1. Pull down the new site repo
2. `yarn add @shiftcommerce/shift-react-component`
3. In a page, add:
```
import { SearchFilters } from '@shiftcommerce/shift-react-components'
```
4. In a `render()` add `<SearchFilters />`
5. You should now see the error in the browser

**Comment by simonireilly** [comment]
@jameskirkby correct :+1:  This needs to rendered with the `InstantSearch` block. Under the hood it is doing a lot of things - passing React context, setting up Providers etc. 

Any compoents wich require access to the `InstantSearch` store need to be within it's block - not sure we have any way to change this behaviour.

**Comment by owensmooth** [comment]
@jameskirkby have you tried importing algoliaReduxWrapper from shift-next?
e.g
```
import { algoliaReduxWrapper } from '@shiftcommerce/shift-next'

export default algoliaReduxWrapper(CategoryPage, CategoryPage)
```

**Comment by jameskirkby** [comment]
@owensmooth @simonireilly I think part of my issue was understanding the ecosystem here. I thought the plan was that `shift-react-components` could operate without `shift-next`, however it looks like if you want to use `SearchFilters`, you have to use `shift-next`?

Does that mean you have to use `next js` if you want to use `shift-react-components`? You can't use another react-based framework?

**Comment by owensmooth** [comment]
@jameskirkby not at all, if you want to use the component in another react-based framework you will need to wrap the app in InstantSearch for it to work, just follow the react-instantsearch docs.

**Comment by jameskirkby** [comment]
Good to know, probably worth documenting this somewhere

**Comment by jameskirkby** [comment]
Probably worth leaving this open until there is some docs for this
